tools/kallisto_calc_expression.cwl

```diff
@@ -33,6 +39,8 @@ inputs:
   reads1: File
   reads2: File
   SampleID: string
+  std_dev: long
+  avg_frag_len: int
```

Comment:
These are optional in the workflow and only seem to be used if the reads2 file is null. These should be optional here as well.